### PR TITLE
remove oc dependency on internal route types

### PR DESCRIPTION
**Pull request by sanchezl**

### CODE REVIEW

File: pkg/oc/lib/newapp/app/pipeline.go [hunk 1]
```diff
@@ -432,28 +431,6 @@ func addService(containers []corev1.Container, objectMeta metav1.ObjectMeta, sel
 	return svc
 }
 
-// AddRoutes sets up routes for the provided objects.
-func AddRoutes(objects Objects) Objects {
```
Comment (deads2k) [inline pkg/oc/lib/newapp/app/pipeline.go]: This looks weird.  What are you doing instead?

Comment (sanchezl) [reply pkg/oc/lib/newapp/app/pipeline.go]: I didn't see where this was used. I reverted my removal.

File: pkg/oc/cli/newapp/newapp.go [hunk 2]
```diff
@@ -420,7 +419,7 @@ func (o *AppOptions) RunNewApp() error {
 		{Version: "v1", Kind: "Pod"}:                                    true,
 		{Group: buildapi.GroupName, Version: "v1", Kind: "BuildConfig"}: true,
```
Comment (deads2k) [inline pkg/oc/cli/newapp/newapp.go]: fix up these other ones while you're here.

**Comment by deads2k** [comment]
if it works

/lgtm

**Pull request merged**